Server/src/main/webapp/index.html

```diff
@@ -4,213 +4,49 @@
-                <p class="feedback-counter">100</p>
-                <button class = "feedback-button"><img class="icon" src="downward.png"></button>
-                <p class="feedback-counter">50</p>
-                <button class="feedback-button"><img class="icon" src="forum.png"></button>
-                <p class="feedback-counter">30</p>
-            </div>
-        </div>
-    </div>
-    <div class="twit">
-        <img class="user-icon" src="Logo.png">
-        <div class="content">
-            <p class="date">01.03.2020</p>
-            <p class = "text">In my opinion you are bad guy!!!!</p>
-            <p class="name">Andrei</p>
-            <p class="tags">#truth#absolutely#up</p>
-            <div class="feedback">
-                <button class = "feedback-button"><img class = "icon" src="upward.png"></button>
-                <p class="feedback-counter">100</p>
-                <button class = "feedback-button"><img class="icon" src="downward.png"></button>
-                <p class="feedback-counter">50</p>
-                <button class="feedback-button"><img class="icon" src="forum.png"></button>
-                <p class="feedback-counter">30</p>
-            </div>
-        </div>
-    </div>
-    <div class="twit">
-        <img class="user-icon" src="Logo.png">
-        <div class="content">
-            <p class="date">01.03.2020</p>
-            <p class = "text">In my opinion you are bad guy!!!!</p>
-            <p class="name">Andrei</p>
-            <p class="tags">#truth#absolutely#up</p>
-            <div class="feedback">
-                <button class = "feedback-button"><img class = "icon" src="upward.png"></button>
-                <p class="feedback-counter">100</p>
-                <button class = "feedback-button"><img class="icon" src="downward.png"></button>
-                <p class="feedback-counter">50</p>
-                <button class="feedback-button"><img class="icon" src="forum.png"></button>
-                <p class="feedback-counter">30</p>
-            </div>
-        </div>
-    </div>
+<div class="twits" id="container">
+    <template id="twit-template">
+        <div class="twit" id="">
+            <img class="user-icon" data-target="user-icon" src="Logo.png">
+            <div class="content">
+                <p class="date" data-target="createdAt"></p>
+                <p class = "text" data-target="description"></p>
+                <img class="twit-photo" data-target="photoLink">
+                <p class="name" data-target="author"></p>
+                <p class="tags" data-target="hashTags"></p>
+                <div class="twit-buttons">
+                    <button class="button-delete">Delete</button>
+                    <button class="button-refactor">Refactor</button>
+                </div>
+                <div class="feedback">
+                    <button class = "feedback-button"><img class = "icon" src="upward.png"></button>
```

Comment:
` <button class="feedback-button"><img class="icon" src="upward.png"></button>`
Make everything consistent. Why in line 38, for example, u write `class="..."` and in line 44 `class = "..."`